src/base/bittorrent/torrenthandle.h

```diff
@@ -417,6 +423,10 @@ namespace BitTorrent
         QQueue<EventTrigger> m_moveFinishedTriggers;
         int m_renameCount;
 
+        // Until libtorrent provide an "old_name" field in `file_renamed_alert`
+        // we will rely on this workaround to remove empty leftover folders
+        QHash<LTFileIndex, QVector<QString>> m_oldPath;
```

Comment:
I would use `int` here so not have allowed the leak of these internal types to outside.

Reply:
`LTFileIndex` is declared in `private` section so it won't be visible to outside of the class, are you sure?

Reply:
Okay, whatever...